pkg/log/log.go

```diff
@@ -0,0 +1,148 @@
+// Copyright 2019, Verizon Media Inc.
+// Licensed under the terms of the 3-Clause BSD license. See LICENSE file in
+// github.com/yahoo/k8s-athenz-istio-auth for terms.
+package log
+
+import (
+	"io"
+	"os"
+	"path/filepath"
+
+	"github.com/sirupsen/logrus"
+	"gopkg.in/natefinch/lumberjack.v2"
+)
+
+var log *logrus.Logger
+
+// InitLogger initializes a logger object with log rotation
+func InitLogger(logFile, level string) {
```

Comment:
can we have an empty string as a possible stdout only logging mode? would be useful for tests without having the need to initialize to `/tmp/k8s-athenz-istio-auth.log`